IdentityCore/src/cache/accessor/MSIDLegacyTokenCacheAccessor.h

```diff
@@ -33,6 +33,7 @@
 @protocol MSIDRequestContext;
 @class MSIDLegacyAccessToken;
 @class MSIDOauth2Factory;
+@class MSIDKeyedArchiverSerializer;
```

Comment:
nit: is this one necessary?

Reply:
fixed, also removed `MSIDLegacyRefreshToken` and `MSIDOauth2Factory` which are also unnecessary in this file :)